tmol/chemical/constants.py

```diff
@@ -16,3 +16,10 @@
 # computing this distance exactly is slow) without affecting the score
 
 MAX_SIG_BOND_SEPARATION = 6
+
+# MAX_PATHS_FROM_CONNECTION:
+#
+# The maximum number of paths coming from a particular block type's connection.
+# Currently this is 13 - 1 for the connection atom itself, 3 for the up-to-3 neighbors
```

Comment:
Yeah, that was a poor use of a dash.

I've updated with your suggestion and also added your generalized solution to the comment in case we want to utilize that in the future.